.github/workflows/continuous-integration.yml

```diff
@@ -10,63 +10,69 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+
     steps:
       - name: "Checkout"
         uses: actions/checkout@master
 
-      - name: "Install PHP 7.4"
-        run: |
-          sudo add-apt-repository ppa:ondrej/php
-          sudo apt-get update
-          sudo apt-get install php7.4 php7.4-mbstring php7.4-xml
+      - name: "Set default PHP version"
+        run: sudo update-alternatives --set php /usr/bin/php${{ matrix.php-version }}
 
       - name: "Validate composer.json"
-        run: php7.4 $(which composer) validate --strict
+        run: composer validate --strict
 
+      - name: "Determine Composer cache directory"
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
       - name: "Cache dependencies"
-        uses: actions/cache@v1
-        id: cache
+        uses: actions/cache@v2
         with:
-          path: ~/.composer/cache/files
+          path: ${{ steps.composer-cache.outputs.dir }}
           key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-composer-
+          restore-keys: ${{ runner.os }}-composer-
```

Comment:
Here is how to use `actions/cache`

Reply:
Nice! I think I only saw the documentation for the v1 so far.